addons/io_scene_gltf2_msfs/io/msfs_multi_export.py

```diff
@@ -101,7 +101,7 @@ def execute(self, context):
                         if lod.enabled:
                             lod_files[lod.file_name] = lod.lod_value
 
-                    lod_files = sorted(lod_files.items(), reverse=True)
+                    lod_files = sorted(lod_files.items(), reverse=False)
```

Comment:
May as well remove the `reverse` option since it's default to false. Also, this doesn't fix the case for the last option having `minSize`, when it shouldn't

Reply:
Indeed I missed that one
I removed the reverse and added a conditionnal statement about the minSize for the last lod